src/amo/components/SecondaryHero/index.js

```diff
@@ -28,7 +52,11 @@ export const SecondaryHeroBase = ({ shelfData }: Props) => {
           {module.description}
         </div>
         {module.cta && (
-          <a className="SecondaryHero-module-link" href={module.cta.url}>
+          <a
+            className="SecondaryHero-module-link"
+            href={makeCallToActionURL(module.cta.url)}
+            {...getLinkProps(module.cta)}
+          >
```

Comment:
You're right, this should be a `Link`, I'll change that. I added the `rel` because I originally had `target="_blank"` hardcoded for the `<a>` tag, and eslint was complaining that it also wanted the `rel` defined, but I can see how if we change it to `Link` then that will just be taken care of.